File: Placements.DataAccess/placement/Models/Companyindustry.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Companyindustry
{
    public long Id { get; set; }

    public long? CompanyId { get; set; }

    public long? IndustryId { get; set; }

    public ulong IsDeleted { get; set; }

    public virtual Companydatum? Company { get; set; }

    public virtual Industry? Industry { get; set; }
}

File: web-app/src/app/services/types/Companyindustry.ts
import { Industry } from "src/app/features/company-configuration/company-config/companies/companies-model";
import { Companydatum } from "./Companydatum";

export type Companyindustry = {
    Id: number;
    CompanyId?: number;
    IndustryId?: number;
    IsDeleted: boolean;
    Company: Companydatum[];
    Industry: Industry[];
}